### ISSUE TITLE
Install via PHIVE fails due to the GPG signing key not (or no longer) being available

### ISSUE THREAD

**Issue by oliverklee** [phpmd/phpmd#1222]
- PHPMD version: 2.15.0
- PHP Version: 8.3.16
- Installation type: PHAR via PHIVA
- Operating System / Distribution & Version: Ubuntu 24.04

### Current Behavior

```
Downloading https://github.com/phpmd/phpmd/releases/download/2.15.0/phpmd.phar
Downloading https://github.com/phpmd/phpmd/releases/download/2.15.0/phpmd.phar.asc
Downloading key 9093F8B32E4815AA
Warning:  Failed with error code 0: No mirror yielded any result. Giving up.
Trying to connect to keyserver.ubuntu.com (185.125.188.27)
Downloading https://keyserver.ubuntu.com/pks/lookup?op=get&options=mr&search=0x9093F8B32E4815AA
Trying to connect to keyserver.ubuntu.com (185.125.188.26)
Downloading https://keyserver.ubuntu.com/pks/lookup?op=get&options=mr&search=0x9093F8B32E4815AA
Trying to connect to keyserver.ubuntu.com (2620:2d:4000:1007::d43)
Error:    Request failed: Couldn't connect to server (while requesting https://keyserver.ubuntu.com/pks/lookup?op=get&options=mr&search=0x9093F8B32E4815AA)
Trying to connect to keyserver.ubuntu.com (2620:2d:4000:1007::70c)
Error:    Request failed: Couldn't connect to server (while requesting https://keyserver.ubuntu.com/pks/lookup?op=get&options=mr&search=0x9093F8B32E4815AA)
Warning:  Failed with error code 404: Key not found on keyserver
Trying to connect to pgp.mit.edu (18.9.60.141)
Downloading https://pgp.mit.edu/pks/lookup?op=get&options=mr&search=0x9093F8B32E4815AA
Warning:  Failed with error code 502: Server reported an error
Error:    PublicKey 9093F8B32E4815AA not found on key servers
```

### Expected Behavior

The PHPMD PHAR is installed, and there are no error messages.

### Steps To Reproduce:

Use this `.phive/phars.xml`:

```
<phive xmlns="https://phar.io/phive">
  <phar name="phpmd" version="^2.15.0" installed="2.15.0" location="./.phive/phpmd" copy="false"/>
</phive>
```

Run this:

```bash
phive --no-progress install --trust-gpg-keys E7A745102ECC980F7338B3079093F8B32E4815AA
```

## Checks before submitting
* [x] Be sure that there isn't already an issue about this. See: [Issues list](https://github.com/phpmd/phpmd/issues)
* [x] Be sure that there isn't already a pull request about this. See: [Pull requests](https://github.com/phpmd/phpmd/pulls)
* [x] I have added every step to reproduce the bug.
* [x] If possible I added relevant code examples.
* [x] This issue is about 1 bug and nothing more.
* [x] The issue has a descriptive title. For example:  "JSON rendering failed on Windows for filenames with space".

**Comment by julianhofmann** [comment]
`pgp.mit.edu` has already been removed in October (https://github.com/phar-io/phive/commit/9dee3083e1c3c725da498a0a0c34bb8db40762f3).
Sadly, no new release has been tagged since this change/fix...

**Comment by AJenbo** [comment]
@ravage84 @tvbeek is this something one of you are able to solve?
